content/en/docs/contribute/intermediate.md

```diff
@@ -465,10 +475,24 @@ most up-to-date version of that branch.
     **Details** link goes to a staged version of the Kubernetes website with
     your changes applied. This is how reviewers will check your changes.
 
-9.  If you notice that more changes need to be made, or if reviewers give you
-    feedback, address the feedback locally, then repeat step 4 - 6 again,
-    creating a new commit. The new commit is added to your pull request and the
-    tests run again, including re-staging the Netlify staged site.
+9.  When more changes need to be made, address the feedback locally and `amend`
+your original commit.
+
+    ```bash
+    git commit -a --amend
+    ```
+
+    - `-a`: commit all changes
+    - `--amend`: amend the previous commit, rather than creating a new one
+
+    An editor will open so you can update your commit message if necessary.
+
+    If you use `git commit -m` as in Step 4, you will create a new commit rather
+    than amending changes to your original commit. Creating a new commit means
+    you must squash your commits before your pull request can be merged.
+
+    The commit is added to your pull request and the tests run again, including
```

Comment:
Is that right? I think that local tests would run at this point (if Hugo is watching for them). Netlify won't detect changes until the contributor pushes to their fork.

Reply:
That is the original wording, which I will clarify.

Reply:
Ah. Thanks!

Reply:
Thanks - I'll clarify.